dealroom_firestore_connector/__init__.py

```diff
@@ -177,13 +179,14 @@ def get_history_doc_refs(
         >>> doc_refs = get_history_refs(db, "dealroom.co")
     """
 
-    collection_path = "history"
-    collection_ref = db.collection(collection_path)
+    collection_ref = db.collection(HISTORY_COLLECTION_PATH)
 
     if str(finalurl_or_dealroomid).isnumeric():
         query_params = ["dealroom_id", "==", int(finalurl_or_dealroomid)]
     else:
-        query_params = ["final_url", "==", finalurl_or_dealroomid]
+        # Extract the final_url in the required format, so we can query the collection with the exact match.
+        final_url = extract(finalurl_or_dealroomid)
```

Comment:
the `extract` function can raise an exception if the format of the url that is being parsed is incorrect. Like: http://sadasd...something. Maybe surround it on a try/except?

Reply:
correct, will do